P/polymake/build_tarballs.jl

```diff
@@ -100,17 +109,17 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("CompilerSupportLibraries_jll")
-    Dependency("FLINT_jll")
-    Dependency("GMP_jll")
-    Dependency("MPFR_jll")
-    Dependency("PPL_jll")
-    Dependency(PackageSpec(name="Perl_jll", uuid="83958c19-0796-5285-893e-a1267f8ec499", version=v"5.30.3"))
-    Dependency("bliss_jll")
-    Dependency("boost_jll")
-    Dependency("cddlib_jll")
-    Dependency("lrslib_jll")
-    Dependency("normaliz_jll")
+    Dependency("CompilerSupportLibraries_jll"),
+    Dependency("FLINT_jll"),
+    Dependency("GMP_jll"),
+    Dependency("MPFR_jll"),
```

Comment:
This now conflicts with master, so I am afraid a rebase is now necessary after all (or just don't add the commas here, it doesn't seem to be necessary?)